repos/system_upgrade/common/actors/checkcustommodifications/libraries/checkcustommodifications.py

```diff
@@ -0,0 +1,41 @@
+from leapp import reporting
+from leapp.libraries.stdlib import api
+from leapp.models import CustomModifications
+
+FMT_LIST_SEPARATOR = "\n    - "
+
+
+def _create_report(msgs, report_type, component=None, hint=None):
+    filtered_msgs = [m for m in msgs or [] if m.type == report_type and (not component or m.component == component)]
+    if not filtered_msgs:
+        # Nothing to report
+        return
+    discovered_files = ''.join(
+            ['{sep}{filename}{actor}'.format(sep=FMT_LIST_SEPARATOR,
+                                             filename=m.filename,
+                                             actor=' ({} Actor)'.format(m.actor_name) if m.actor_name else '')
```

Comment:
Reading the report msgs like: `(quagga_to_frr Actor)` seems weird to me. Let's switch the order and add a colon as a separator: 
```suggestion
                                             actor=' (Actor: {})'.format(m.actor_name) if m.actor_name else '')
```